swan-cern/files/swan_config_cern.py

```diff
@@ -187,6 +197,23 @@ def _init_eos_containers(self, eos_secret_name):
         # assigning pod spec containers
         self.pod.spec.containers = pod_spec_containers
 
+    def _gpu_enabled(self):
```

Comment:
This method already comes with the `swan` chart (the file will be evaluated before), so I don't think it's required. Did you check?

Reply:
right,fixed